src/pkg/selectors/exchange.go

```diff
@@ -492,20 +493,18 @@ const (
 	ExchangeMail          exchangeCategory = "ExchangeMail"
 	ExchangeMailFolder    exchangeCategory = "ExchangeMailFolder"
 	ExchangeUser          exchangeCategory = "ExchangeUser"
-	// append new data cats here
-
-	// filterable topics identified by exchange
-	ExchangeFilterMailSender         exchangeCategory = "ExchangeFilterMailSender"
-	ExchangeFilterMailSubject        exchangeCategory = "ExchangeFilterMailSubject"
-	ExchangeFilterMailReceivedAfter  exchangeCategory = "ExchangeFilterMailReceivedAfter"
-	ExchangeFilterMailReceivedBefore exchangeCategory = "ExchangeFilterMailReceivedBefore"
-	ExchangeFilterContactName        exchangeCategory = "ExchangeFilterContactName"
-	ExchangeFilterEventOrganizer     exchangeCategory = "ExchangeFilterEventOrganizer"
-	ExchangeFilterEventRecurs        exchangeCategory = "ExchangeFilterEventRecurs"
-	ExchangeFilterEventStartsAfter   exchangeCategory = "ExchangeFilterEventStartsAfter"
-	ExchangeFilterEventStartsBefore  exchangeCategory = "ExchangeFilterEventStartsBefore"
-	ExchangeFilterEventSubject       exchangeCategory = "ExchangeFilterEventSubject"
-	// append new filter cats here
+
+	// data contained within details.ItemInfo
+	ExchangeInfoMailSender         exchangeCategory = "ExchangeInfoMailSender"
```

Comment:
just double checking: we never need to deserialize these values to display to the user right? Like we never show the selector that was used for a backup with backup list?

Reply:
Yes, this is entirely under the hood, and scoped to the current.  No end user display, no re-use after retrieval.